test/tf/vsphere/scripts/preload_vars.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
Is there any way we can keep this pure Terraform with it handling all variable prompting, or pulling from the env? Trying to keep the CI consistent in setup/teardown and want to go all Terraform if possible.

Reply:
This script was just for CLI testing. We will need to set some env to get acceptance tests working (namely the configuration for vSphere access), and I'd prefer to keep most of the details of the lab in CI and out of configuration to make it portable. We can get rid of the script though, as most of this can be gleaned from config (I could just move the documentation for what is needed to comments or another file).